src/components/TransactionSender.tsx

```diff
@@ -161,7 +170,11 @@ class TransactionSender extends React.Component<Props, State> {
 
     try {
       if (await requiresRemoteSignatures(horizon, signedTx, account.publicKey)) {
-        await this.submitTransactionToMultisigService(signedTx)
+        if (await isStellarGuardProtected(horizon, account.publicKey)) {
```

Comment:
[Minor] Let's flatten that `if` chain:

```ts
if (await isStellarGuardProtected(horizon, account.publicKey)) {
  // ...
} else if (await requiresRemoteSignatures(horizon, signedTx, account.publicKey)) {
  // ...
} else {
  // ...
}
```

Less nesting = better-to-maintain code